src/Lifi.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
+/*eslint max-len: ["error", { "ignoreComments": true }]*/
```

Comment:
I would rather do that in the global eslint configuration? We don't need maxlength errors on comments anywhere I would say

Reply:
good point, completely forgot that we have one...